server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Authentication -> WebAuthn Policy` tab
+- configure items and push `Save` button
+
+The configurable items and their description are shown just below.
+
+|===
+|Configuration|Description
+
+|Relying Party Entity Name
+|Human-readable server name as WebAuthn Relying Party. This is mandatory configuration item and applied to the operation of registering WebAuthn authenticator. The default setting is "keycloak".
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#dictionary-pkcredentialentity[WebAuthn Specification].
+ 
+|Signature Algorithms
+|It tells the WebAuthn authenticator which signature algorithms be used for the https://www.w3.org/TR/webauthn/#public-key-credential[Public Key Credential] that can be used to sign and verify the https://www.w3.org/TR/webauthn/#authentication-assertion[Authentication Assertion]. Multiple algorithms can be specified. If no algorithm is specified, https://tools.ietf.org/html/rfc8152#section-8.1[ES256] is adapted. The default setting is ES256. This is optional configuration item and applied to the operation of registering WebAuthn authenticator.
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#dictdef-publickeycredentialparameters[WebAuthn Specification].
+
+|Relying Party ID
+|This is ID as WebAuthn Relying Party and determines the scope of Public Key Credentials. It must be origin's effective domain. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no entry is entered, the host part of the base URL of {project_name}'s server is adapted.
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#rp-id[WebAuthn Specification].
+
+|Attestation Conveyance Preference
+|It tells the WebAuthn API implementation on the browser (https://www.w3.org/TR/webauthn/#webauthn-client[WebAuthn Client]) the preference of how to generate an Attestation Statement. However, {project_name} does not verify Attestation Statement so that only "none" option can be selected. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "none".
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#attestation-convey[WebAuthn Specification].
+
+|Authenticator Attachment
+|It tells the WebAuthn Client an acceptable attachment pattern of a WebAuthn authenticator. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no option is selected, the WebAuthn Client does not consider the attachment pattern.
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#enumdef-authenticatorattachment[WebAuthn Specification].
```

Comment:
Try:  For more details, see the https://www.w3.org/TR/webauthn/#enumdef-authenticatorattachment[WebAuthn Specification].

Reply:
Thank you. I'll fix this sentence as you suggested.